client/shipping-label/index.js

```diff
@@ -46,8 +46,8 @@ export default ( { formData, labelsData, paperSize, storeOptions, paymentMethod
 					},
 					destination: {
 						values: formData.destination,
-						isNormalized: false,
-						normalized: null,
+						isNormalized: ! ! formData.destination_normalized,
```

Comment:
We prefer `Boolean(value)` instead of `!!` (old discussion, I don't know how strict have we been so far).